src/System.Private.ServiceModel/src/Resources/Strings.resx

```diff
@@ -1932,4 +1932,22 @@
   <data name="Xml_InvalidNodeType" xml:space="preserve">
     <value>'{0}' is an invalid XmlNodeType.</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="SFxRpcMessageMustHaveASingleBody" xml:space="preserve">
+    <value>RPC Message {1} in operation {0} must have a single MessageBodyMember.</value>
+  </data>
+  <data name="SFxBodyObjectTypeCannotBeInherited" xml:space="preserve">
+    <value>Type {0} cannot inherit from any class other than object to be used as body object in RPC style.</value>
```

Comment:
`RPC style` seems to be a term that people might use. For example, [this article](https://www.javatpoint.com/difference-between-rpc-and-document) uses `RPC style` in this way. I'm going to leave the message as is. Thanks for the suggestion, @StephenBonikowsky .